core/edit/edit_navigation_steps.py

```diff
@@ -39,21 +35,21 @@ class Actions:
     def perform_navigation_steps(steps: list[NavigationStep]):
         """Navigate by a series of steps"""
         for step in steps:
-            match step.direction:
-                case "up":
-                    repeat_action(actions.edit.up, step.count)
-                case "down":
-                    repeat_action(actions.edit.down, step.count)
+            match step.type:
+                case "wordLeft":
+                    repeat_action(actions.edit.word_left, step.count)
```

Comment:
The word variants of this command suffer from the same inconsistent behavior that led me to add the delay for the recently added repeatable commands. I see this inconsistency reliably in VSCode on windows, though in practice I really don't use it
 https://github.com/talonhub/community/blob/ac4ea587ab5ef85ddd0d37940d2c96e76d0bff1d/core/edit/edit_command.py#L12